src/pages/home/sidebar/GlobalNavigation/GlobalNavigationMenuItemList.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'underscore';
+import React, {useContext} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import GlobalNavigationMenuItem from './GlobalNavigationMenuItem';
+import {SidebarNavigationContext} from '../SidebarNavigationContext';
+import defaultTheme from '../../../../styles/themes/default';
+
+const propTypes = {
+    /** Menu items to be rendered on the list */
+    menuItems: PropTypes.arrayOf(
+        PropTypes.shape({
+            /** An icon element displayed on the left side */
+            icon: PropTypes.elementType,
+
+            /** Text label */
+            text: PropTypes.string.isRequired,
+
+            /** A callback triggered when this item is selected */
+            onSelected: PropTypes.func.isRequired,
+        }),
+    ).isRequired,
+
+    /* Styles for the menu items list */
+    style: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
```

Comment:
@adamgrzybowski What is the reason for passing the style as a prop as opposed to defining the style inside the `GlobalNavigationMenuItemList` component?

Reply:
I'm also not sure if we need to create the `GlobalNavigationMenuItemList` itself as opposed to adding the list of menu items inside `GlobalNavigation.js` directly

Reply:
If I remember correctly, Wojtek copied the list component from another place when he started working on this POC. It's possible that some props are not necessary or even maybe we merge this into one component as you suggested